apps/wallet/src/ui/app/components/receipt-card/index.tsx

```diff
@@ -161,7 +161,7 @@ function ReceiptCard({ txDigest }: TxResponseProps) {
                         {assetCard}
                     </div>
 
-                    {transferMeta[transferType].address && (
+                    {transferMeta[transferType].address ? (
```

Comment:
Why is this change needed?

Reply:
Oh, remove extra 0. 
<img width="401" alt="Screenshot 2023-01-14 at 10 23 41 AM" src="https://user-images.githubusercontent.com/8676844/212551264-64ee360b-7321-4fa6-93d2-5e15498eb1b0.png">